Step 1: Custom DOM manipulation challenge

fatihtutar opened the pull request:
ex 1 part 1

fatihtutar committed:
ex 1 part 1

bitsagarob reviewed (changes requested):
very good, now also make the todo list bonus exercise from this file

fatihtutar committed:
ex 1 part 2

bitsagarob reviewed (approved):
Great job! Now onto the original homework, this will be more difficult. But you'll be able to!

fatihtutar committed:
Step 1: Custom DOM manipulation challenge

I have been stucked in the problem below since 2 days googled it a lot but couldn"t find a solution. Console gives "Uncaught TypeError: Cannot read property 'appendChild' of null" warning.
/ Add it to the page
  listContainer.appendChild(listElement);

fatihtutar committed:
Update app.js

fatihtutar committed:
index.html

segersrobbertKlarrio commented on Week1/homework/app.js:
```diff
@@ -1,11 +1,123 @@
+  const listElement = document.createElement('ul');
+
+  // Add it to the page
+  listContainer.appendChild(listElement);
```
if it appears the `listContainer` variable has no value. Verify by console logging that variable and fix the probable issue at line 34 where you fetch the element

fatihtutar replied:
It seems that the problem is solved. Thanks

segersrobbertKlarrio commented on Week1/homework/index.html:
```diff
@@ -1 +1,14 @@
+    <title>Book Blog</title>
+  </head>
+  <body>
+    <div id= "books"></div>
```
there is a typo here

fatihtutar committed:
Update app.js

fatihtutar committed:
Update app.js

bitsagarob commented on Week1/homework/app.js:
```diff
@@ -1,11 +1,123 @@
+  // Set up a loop that goes through the items in listItems one at a time
+  const numberOfListItems = bookTitles.length;
+  function bookList() {
+  for (let i = 0; i < numberOfListItems; ++i) {
```
indentation of for loop

bitsagarob reviewed (approved):
Great job, beautiful code & commenting. Try to finish the books exercise!
